proto/tendermint/types/types.proto

```diff
@@ -76,8 +76,31 @@ message Data {
   // NOTE: not all txs here are valid.  We're just agreeing on the order first.
   // This means that block.AppHash does not include these txs.
   repeated bytes txs = 1;
+  reserved 2, 3, 4;
+  // field number 2 is reserved for intermediate state roots
+  // field number 3 is reserved for evidence
+  // field number 4 is reserved for blobs
+
+  // SquareSize is the number of rows or columns in the original data square.
+  uint64 square_size = 5;
+
+  // Hash is the root of a binary Merkle tree where the leaves of the tree are
+  // the row and column roots of an extended data square. Hash is often referred
+  // to as the "data root".
+  bytes data_root_hash = 6;
```

Comment:
Think this should be `hash`. I'm seeing a test failure in app where there is a testdata json object and test yielding: `error unmarshal JSON block response: unknown field "hash" in types.Data`